lib/Driver/Compilation.cpp

```diff
@@ -206,15 +208,21 @@ namespace driver {
     /// Jobs that incremental-mode has decided it can skip.
     CommandSet DeferredCommands;
 
-    /// Jobs in the initial set with Condition::Always, or lacking existing
+    /// Jobs in the initial set with Condition::Always, or lacking an existing
     /// .swiftdeps files.
```

Comment:
Micro-nit: In "lacking existing .swiftdeps files", files is plural. If you're going to put an "an" in there, drop the "s" on "files".

Reply:
Thanks for the catch! I had meant to not change that comment at all, since the fix is in my other PR. And thanks for looking at this PR so promptly.